cli/src/Morphir/Web/DevelopApp.elm

```diff
@@ -1344,6 +1353,60 @@ viewHome model packageName packageDef =
                 , Element.Keyed.row ([ width fill, height <| fillPortion 23, scrollbars ] ++ listStyles) [ ( "definitions", viewDefinitionLabels (model.homeState.selectedModule |> Maybe.map Tuple.second) ) ]
                 ]
+
+        homeTabs =
+            let
+                col elements =
+                    column
+                        [ height fill
+                        , width fill
+                        , scrollbars
+                        ]
+                        elements
+
+                leafModules : ModuleName -> List ModuleName
+                leafModules moduleName =
+                    packageDef.modules |> Dict.keys |> List.filter (\l -> List.concat l |> String.join "." |> String.startsWith (List.concat moduleName |> String.join "."))
+
+                summary =
+                    let
+                        numberOfModules =
+                            packageDef.modules |> Dict.keys |> List.length
+
+                        numberofValuesAndTypes acmoduledef moduleName =
+                            row [ width fill ] [ text <| (moduleName |> List.map Name.toTitleCase |> String.join ".") ++ " : " ++ String.fromInt (acmoduledef.value.values |> Dict.keys |> List.length) ++ " definition(s) and " ++ String.fromInt (acmoduledef.value.types |> Dict.keys |> List.length) ++ " type(s)." ]
+                    in
+                    case model.homeState.selectedModule of
+                        Just ( modulePath, moduleName ) ->
+                            column [ spacing (Theme.smallSpacing model.theme) ]
+                                (leafModules moduleName
+                                    |> List.map
+                                        (\mn ->
+                                            case Dict.get mn packageDef.modules of
+                                                Just acmd ->
+                                                    numberofValuesAndTypes acmd mn
+
+                                                Nothing ->
+                                                    Element.none
+                                        )
+                                )
+
+                        Nothing ->
+                            row [ width fill ] [ text <| "This package contains " ++ String.fromInt numberOfModules ++ " modules." ]
+            in
+            TabsComponent.view model.theme
+                { onSwitchTab = UI << SwitchTab
+                , activeTab = model.activeTabIndex
+                , tabs =
+                    Array.fromList
+                        [ { name = "Summary"
+                          , content = el [ padding <| Theme.smallPadding model.theme ] summary
+                          }
+                        , { name = "Dependency Graph"
+                          , content = col [ dependencyGraph model.homeState.selectedModule model.repo ]
+                          }
+                        ]
+                }
```

Comment:
I think this whole section would deserve its own function. Also, make sure you always specify the types of your definitions.